### PR TITLE
build-finish: Add --extension-priority option

### PR DESCRIPTION
**Pull request by alexlarsson**
This lets you set the priority of the extension.

### CODE REVIEW

File: app/flatpak-builtins-build-finish.c [hunk 1]
```diff
@@ -40,6 +40,7 @@ static char **opt_extra_data;
 static char **opt_extensions;
 static char **opt_metadata;
 static gboolean opt_no_exports;
+static int opt_extension_prio = -10000;
```
Comment (mwleeds) [inline app/flatpak-builtins-build-finish.c]: Might be slightly better to use `G_MININT`

File: doc/flatpak-build-finish.xml [hunk 2]
```diff
@@ -341,6 +341,15 @@ key=v1;v2;
             </varlistentry>
 
             <varlistentry>
+                <term><option>--extension-priority=VALUE</option></term>
+
+                <listitem><para>
+                  Set the priority (library override order) of the extension point.
```
Comment (mwleeds) [inline doc/flatpak-build-finish.xml]: Might be worth mentioning "Default is 0. Higher means more prioritization"

**Comment by alexlarsson** [comment]
Added requested changes

**Comment by cgwalters** [comment]
Hey @mwleeds  - this repo now uses Homu: https://github.com/projectatomic/papr#more-details-about-project-atomic-ci-services

Homu takes care of queuing merges.

**Comment by mwleeds** [comment]
@cgwalters oh sorry. I thought it was weird that @rh-atomic-bot hadn't commented but I guess that doesn't happen if the author is already on the whitelist.

The bot doesn't squash for you right?

**Comment by cgwalters** [comment]
The bot does squash fixups.

**Pull request merged**